packages/project-account-switcher/src/presenters/ContentPresenter.test.js

```diff
@@ -13,7 +13,11 @@ describe("project-account-switcher/presenters/ContentPresenter", () => {
         accountTitle: "Accounts",
         accounts: [
           { id: "1", label: "Account 1" },
-          { id: "2", label: "Account 2" }
+          {
+            id: "2",
+            label: "Account 2",
+            image: "http://placekitten.com/g/60/32"
```

Comment:
We actually do place fixtures in `storybook`, here is an example: https://github.com/Autodesk/hig/blob/development/packages/avatar/src/__stories__/stories.js#L1

Reply:
Oh, yes, i actually took the Avatar as an example reference. I found in the current jest of Avatar, it uses images from placekitten.com in Avatar.test.js; and use built-in image in storybook folder for storybook's stories.

Should we always build images in storybook folder and use it for both jest and storybook stories?

Reply:
And any guideline for which image we should add to our package? Is it ok to find some in Autodesk DAM?

Reply:
Hi, just dug the code a little bit, it seems for jest, for most components, we are using images from placekitten.com like "http://placekitten.com/g/32/32/" as test samples, and for storybook related tests: storybook stories and gemini stories, we are using fixtures in storybook folder. Thought we should just keep the similar usage in this component?

Also, directly import png in jest seems not work in our current configuration, thought it may need to modify some resource name mappings for jest?

Reply:
I think the images you used work are fine. As for jest testing we can keep using the placekitten ones since that is in other jest test as well, no need to change it now and for `storybook` images we will continue use fixtures like we are now. Thanks for letting me know as I actually didn't know it was different from `storybook`.